### ISSUE TITLE
Japanese Text Disappears with IME

### ISSUE THREAD

**Issue by Akashdeep-Sarin** [ianstormtaylor/slate#2944]
#### Do you want to request a _feature_ or report a _bug_?
Bug

#### What's the current behavior?
When a user types a text in Japanese, the text seems to disappear.
At fast typing speeds, the bug is easily reproducible.
Microsoft IME is used for Japanese text in Chrome on Windows.

For bugs, you **must** include the following: 

- A JSFiddle that minimally reproduces the issue:
Any sample application including [example on slate.js](https://www.slatejs.org/#/rich-text) shows this behaviour.

- A GIF showing the issue in action.
![japanese_text_issue](https://user-images.githubusercontent.com/16319756/61785800-1672ea00-ae2a-11e9-9d8c-20fcdb10ce0f.gif)

- Information about your OS, browser, Slate version, etc.
OS: Windows 10
Browser: Chrome
Slate Version: 0.45 (and 0.47.4(latest))

#### What's the expected behavior?

Text should come up as soon as it is typed... Something like as in MS word....
Also being reflected as a part of the Document object in parallel...
![japanese_text_issue_ms_word](https://user-images.githubusercontent.com/16319756/61788635-2ee60300-ae30-11e9-8b93-a131e31f5c5e.gif)

**Comment by milkman4** [comment]
This is a known issue. Here's a similar one 2368

**Comment by ianstormtaylor** [comment]
I believe that this may be fixed by https://github.com/ianstormtaylor/slate/pull/3093, which has changed a *lot* of the logic in Slate and `slate-react` especially. I'm going to close this out, but as always, feel free to open a new issue if it persists for you. Thanks for understanding.
